Updated Contactor Driver Schematic

manthanand opened the pull request:
# Quality Assurance Checklist
To make reviews more efficient, please make sure the board meets the following standards and check everything off once the board meets the quality check. Once everything has been checked, the assigned reviewers will begin the review process. _Edit this description to check off the list._

There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions.

## Schematic Level Requirements 
- [x] Is proper noise resistance given to all peripheral devices (bypass caps and coils/ferrites)?  
- [x] Is proper ESD protection given to all MCU input pins (zener diodes)? You could also write the input pin rating for the MCU on the schematic.  
- [x] Is proper power protection given to peripheral devices (zener diodes)?  
- [x] Are peripheral units used properly (reading datasheet)?  
- [x] Are testing points added at useful places?  
- [x] Is there proper short to GND protection at MCU outputs (inline resistors)?  
- [x] Do ADC inputs have caps (appropriately sized to prevent RC delay)?  
- [x] Are ADC inputs biased so there is room above expected value to determine if value is being overflowed?  
- [x] Are LEDs located at useful places (comm, power, debugging, extra GPIO)? They should be placed in locations   
- [x] Are parts chosen easy to collect?  
- [x] Are parts chosen easy to solder?  
- [x] Is there reverse polarity protection on inputs?  
- [x] Do parts in the schematic match the actual part datasheet?

### Components
- [x] Do footprints match the schematic?
    - Make sure pins on the footprint correspond to pins in the schematic 
    - (e.g. counterclockwise pins on footprint and alternating pins on schematic is *bad*)
- [x] Custom footprints have been double checked with the datasheet.
- [x] Footprints are consistent across the PCB: *i.e. all 0805 for Resistors/Capacitors*
- [ ] Pin 1 of the footprint is labeled in some way. 
- [ ] Are LED's in easy to see places? 
- [ ] Are test points in easy to reach places?  
- [ ] Are critical paths of switching converters as small as possible?  


## Other Comments
Please Just review schematic.

manthanand committed:
Updated Contactor Driver Schematic

manthanand committed:
Added ferrite beads

manthanand committed:
Changed input control logic

manthanand committed:
Changed input logic to normally open

manthanand committed:
Removed PU resistor

manthanand committed:
Moved comments

manthanand committed:
Used INA instead of 2 comparators

manthanand committed:
Updated layout

dimembermatt reviewed (commented):
I'd like to see some comments on the schematic explaining some of the circuits. I think I inferred them properly since I've seen similar from Kevin Mechler's circuits for a not HV/HC application but a less experienced member might be confused. At the very least, a hyperlink textbox to the relevant design doc on sharepoint. This probably applies to some of my boards... Gotta dig out the design docs from my drive again.

Sch:
PowerConn:
0805 1uF cap;looks like the max current is ~8 ARMS at 10MHz. Probably fine. If there was a similar cap for the contactor I'd have questions on the RC constant and thus the inrush current for the capacitor.

DashboardConn:
This looks like an intermediate stage FET to drive the contactor.
What is the charge required at the secondary FET gate to close the contactor? Is the dashboardConn default tied low to keep this disabled?

Same question for Precharge.

Other:
Extra loose wire end at J4.
Why is J4 a 4 pin connector instead of 2 like J5?

Board:
Version history: Should this be on F.Silk layer instead of User.Comments?
3dmodels: Connectors and precharge resistor missing from final model.


Probably also want to update the page settings for PCB metadata (e.g. authors, title, revision, etc)

manthanand committed:
Did layout

manthanand committed:
Created layout

manthanand commented:
Sorry, I didnt see your comments until now:

manthanand commented:
Sorry, I didnt see your comments until now:

1. Added comments
2. Removed caps at output of INA since there is already input capacitance of mosfet. Changed power cap to match datasheet value. Only have 1 input cap at .01u.
3. Dashboard connector is second enable to turn on precharge contactor. If controls is controlling it, then it turns on when it is sent message that BPS is safe. In better implementation. AUX signal of other contactor is tied to Dashboard Conn. It is default low to keep it off.
4. Not sure what you mean by loose wire.
5. J4 is a 2 pin connector?
6. I got rid of board version history cuz there is nothing on it. Also I didn't do it.
7. Updated page settings.

Commit:
Fixed comments

Commit:
Updated UTSVT KiCAD Libararies

Commit:
Merge branch 'master' into v2.0
